Doc/library/time.rst

```diff
@@ -126,6 +126,12 @@ The module defines the following functions and data items:
    This is negative if the local DST timezone is east of UTC (as in Western Europe,
    including the UK).  Only use this if ``daylight`` is nonzero.
 
+   .. note::
```

Comment:
I don't like that the same note is repeated four times.  Maybe we can split the functions and constants in two separate sections and place this note in the constants section once.